genienlp/tasks/almond_task.py

```diff
@@ -603,11 +605,62 @@ def _make_example(self, parts, dir_name=None, **kwargs):
         context = sentence
         answer = sentence # means we calculate self-bleu
         
-        return Example.from_raw(example_id, context, question, answer,
-                                preprocess=self.preprocess_field, lower=False)
+        return Example.from_raw(example_id, context, question, answer, preprocess=self.preprocess_field, lower=False)
     
-    def get_splits(self, root, **kwargs):
-        return AlmondDataset.return_splits(path=os.path.join(root, 'almond'), make_example=self._make_example, **kwargs)
+
+@register_task('translate')
```

Comment:
Should this be "almond_translate"? We already have a number of translation tasks based on WMT. Maybe we should clean that up.